support/README.md

```diff
@@ -14,37 +14,57 @@ First response in an hour (during business time); updates to user until either s
 
 Within a day:
 
-- Email inbox (hi@windmill.engineering) (https://groups.google.com/a/windmill.engineering/forum/#!forum/hi)
+- Email inbox (hi@tilt.dev) (https://groups.google.com/a/tilt.dev/forum/#!forum/hi)
 
-- DMs or Mentions to @windmill_eng
+- DMs or Mentions to (@tilt_dev)[https://twitter.com/tilt_dev]
 
 - All new users should be researched and entered into the CRM. If they’re a
   promising lead, the exterminator should reach out to them.
 
-For any new bugs reported by users, the Exterminator should create a Github
-issue. For any existing bugs reported by users, the Exterminator should note the
-request in the existing Github issue / direct the user to the existing issue and
-ask them to +1 it so we know who to follow up with,
+### Support Tracking
 
-Anyone with a bug to report should slack it to #triage-bugs and, if it turns out
-to be a new bug, file an issue in GitHub . Issues with a bug label are
+We define "issue" as any user support request that we want to track over
```

Comment:
Can we _not_ use "issue" here, since that word is being used to refer to GitHub issues throughout this page.

In particular, we can just call it "user support request". So this section could be:

```
Users requesting support could be because of a bug, a feature request,
a request for docs/examples, or just a general problem with Tilt or using Tilt.

We have two systems for tracking user support requests:

- GitHub issues
- Clubhouse for internal work tracking
```

Reply:
reworked this a bit!